### DISCUSSION TITLE
Versatile Thermostat: Set temperature preset

### DISCUSSION THREAD

**Discussion by johnbur1** [jmcollin78/versatile_thermostat#282] [Q&A] [answered]
I am loving Versatile Thermostat! Incredible work. I have one small question - I don't seem to be able to get 'Versatile Thermostat: Set temperature preset' to do anything.
For example - the thermostat does not change to boost when I run this - what am I missing?
I have set suitable temps for all the modes

service: versatile_thermostat.set_preset_temperature
target:
  entity_id: climate.v_thermostat_back_room
data:
  preset: boost

**Comment by jmcollin78** [comment, marked as answer]
Hello @johnbur1 ,

You are not the first one to fall into this trap. Set Temperature preset is to modify the configured temperature of a preset and not the select the preset.

To select the preset you need to call the climate.set_preset service (the standard service for all climate) and not the versatileThermostat.set_preset_temperature.

I will update the documentation because it seems not clear for everyone.

**Comment by johnbur1** [comment]
Thank you. I thought I had tried that, but clearly not!
